deep.js

```diff
@@ -0,0 +1,55 @@
+/**
+Deep Equal
+
+Determines whether two objects are equal. Works on nested structures.
+
+@param {Object} obj1 can be either an object or array
+@param {Object} obj1 can be either an object or array
+
+@returns {Boolean}
+*/
+
+export {
```

Comment:
Probably should be at the top of the file.

Reply:
put jsdoc above the or inside the function, export at the very top

Reply:
fixed